pyproject.toml

```diff
@@ -52,7 +52,7 @@ readme = "README.md"
 dependencies = [
     "altair>=5.2.0",
     "pandas>=2.0.0",
-    "pydantic>=2.5.2",
+    "pydantic>=1.10.14",
```

Comment:
Not wrong, just sanity checking: do we explicitly want to allow users to install pydantic 2.0 (for e.g. if a user wants to install smartsim in one venv and the dashboard in another), or should this have a `<2` pin?

Reply:
Its passing the unit tests with 2 so in theory it should be fine to leave as is

Reply:
Hmmm that's a good question. I was going to leave it because it does pass the tests with some warnings, but if you think it's a good idea to have a `<2` pin I don't see an issue with that either!

Reply:
On second thought, maybe it's a good idea to try to keep it close to SmartSim. And with the `<2` pin, we don't have to immediately worry when `pydantic 3.0` comes out and makes breaking changes. Unless @ashao knows of a reason to not use the `<2` pydantic pin?

Reply:
Per an offline conversation, we've decided to add a `<2` pin for now to avoid potential breaking changes. We may need to revisit this in the future.